pom.xml

```diff
@@ -579,6 +553,92 @@ limitations under the License.]]></inlineHeader>
     </plugins>
   </build>
   <profiles>
+    <profile>
+      <id>jdk11</id>
+      <activation>
+        <jdk>[9..)</jdk>
+      </activation>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <artifactId>maven-compiler-plugin</artifactId>
+              <version>3.8.1</version>
+              <configuration>
+                <source>${java.version}</source>
```

Comment:
it is within `jdk11` profile, but it uses the default   `<target>${java.version}</target>` that is `    <java.version>1.8</java.version>`. Why is that?

Reply:
because even if we build with jdk11+ we want the code and binaries to be compatible with jdk8.

this patch basically is only to allow users that have jdk11+ to build the sink, I did the same in the other siblings project.
in the Pulsar team we are using JDK11 for every build pipeline.